bibs/Hameln.json

```diff
@@ -21,6 +21,9 @@
             "title": 7
         },
         "baseurl": "http://www.stadtbuecherei.hameln.de",
+        "mediatypes": {
+            "0": "ALL"
+        },
```

Comment:
mediatypes look wrong (see longer comment above), at least `ALL` needs to be removed.

Reply:
Will remove mediatypes, as Hameln does not have them.